### PR TITLE
Fix error if the signature also has an keyword argument 

### PR DESCRIPTION
**Pull request by delijati**
See test :)

### CODE REVIEW

File: fastapi_router_controller/lib/controller.py [hunk 1]
```diff
@@ -100,7 +100,12 @@ def __parse_controller_router(cls):
 
             # replace the class instance with the itself FastApi Dependecy
             signature_parameters[0] = signature_parameters[0].replace(default=Depends(cls))
-            new_signature = signature.replace(parameters=signature_parameters)
+            # set self and after it the keyword args
```
Comment (KiraPC) [inline fastapi_router_controller/lib/controller.py:103]: @delijati Can you explain me with an example what this fix?

File: tests/test_cbv.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import unittest
```
Comment (KiraPC) [inline tests/test_cbv.py:1]: Since tests are in example folder, could you remove it?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: fastapi-router-controller
```
Comment (KiraPC) [inline .github/workflows/ci.yml:1]: Since there already is an action that does this think, could you remove it?

File: tests/test_cbv.py (hunk 2 above)
Comment (delijati) [reply tests/test_cbv.py:1]: The example folder is misleading tests should be in a `tests` folder. But i would still leave the example folder,

File: .github/workflows/ci.yml (hunk 3 above)
Comment (delijati) [reply .github/workflows/ci.yml:1]: Again i would rather see on every push if the tests are failing

File: fastapi_router_controller/lib/controller.py (hunk 1 above)
Comment (delijati) [reply fastapi_router_controller/lib/controller.py:103]: An example is in the tests -> just remove the line above and you will see the error. Btw. take a look into `pytest` for testing

File: tests/test_cbv.py (hunk 2 above)
Comment (KiraPC) [reply tests/test_cbv.py:1]: Ok, let's move the example in test folder.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (KiraPC) [reply .github/workflows/ci.yml:1]: Yes, that's fine. We can have just one action that trigger tests both in push and PR

**Review by KiraPC — APPROVED** [review]

**Pull request merged**